services/QuillLMS/client/app/bundles/Shared/components/shared/__tests__/__snapshots__/postNavigationbanner.test.tsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`PostNavigationBanner it should render 1`] = `
 <DocumentFragment>
   <div
-    class="banner-container green"
+    class="banner-container green green"
```

Comment:
NBD, but it seems a little strange to me that we wound up with an extra `green` here.

Reply:
good catch! I forgot to update the props for this example